src/QueryDemo.sol

```diff
@@ -101,6 +101,12 @@ contract QueryDemo is QueryResponse {
             // Validate that update is not stale
             validateBlockTime(eqr.blockTime, block.timestamp - 300);
 
+            // wormhole time is in microseconds, timestamp is in seconds
+            adjustedBlockTime = eqr.blockTime / 1_000_000;
+            if (adjustedBlockTime <= block.timestamp - 300) {
```

Comment:
I think `validateBlockTime` covers this check (and unfortunately does that same division internally, so we have to do it twice, but :shrug:).

Reply:
updated!